### DISCUSSION TITLE
How can I do to visit my CasaOS ?

### DISCUSSION THREAD

**Discussion by edrun** [IceWhaleTech/CasaOS#593] [Q&A] [answered]
"CasaOS v0.3.6 is running at:
 ─────────────────────────────────────────────────────
 - http://192.168.1.2: (eth0)
 Open your browser and visit the above address."

As above, the  adress "http://192.168.1.2: (eth0)" , where can i find the right port? TKS!

**Comment by Bodyes26** [comment, marked as answer]
you have to navigate to that ip, so put in your address bar 192.168.1.2
the default configuration of casaOS is routed to the port 80, that is the default so you can leave it
